packages/weblets/src/elements/profiles/Profiles.wc.svelte

```diff
@@ -74,7 +74,7 @@
     try {
       const relay = grid.getDefaultUrls(window.env.NETWORK).relay.slice(6);
       const account = await grid.tfchain.createAccount(relay, null);
-      mnemonics.setValue(account.mnemonic, { error: "Please fund your wallet then refresh." });
+      mnemonics.setValue(account.mnemonic);
```

Comment:
I think we still need the error message somehow instead of saying that the balance is not enough. maybe we can remove it from here and update the other error message.

Reply:
Why is this case different than the other one? In the case of new account or using an existing, both will have to fund their accounts, right?

Reply:
yes that's what I mean